lib/lighthouse/benefits_claims/service.rb

```diff
@@ -97,14 +97,16 @@ def submit526(body, lighthouse_client_id = nil, lighthouse_rsa_key_path = nil, o
       # if we're coming straight from the transformation service without
       # making this a jsonapi request body first ({data: {type:, attributes}}),
       # this will put it in the correct format for transmission
-      if body['attributes'].blank?
-        body = {
-          data: {
-            type: 'form/526',
-            attributes: body
-          }
-        }.as_json.deep_transform_keys { |k| k.camelize(:lower) }
-      end
+
+      body = {
+        data: {
+          type: 'form/526',
+          attributes: body
+        }
+      }.as_json.deep_transform_keys { |k| k.camelize(:lower) }
+
+      # TODO: determine why camelcasing is malfunctioning for this field and revisit the below
```

Comment:
mystery answered? perhaps add a comment below as to why the override is needed?

Reply:
Good call. Updated!